admin/javascript/LeftAndMain.Preview.js

```diff
@@ -1,4 +1,20 @@
 (function($) {
+
+	var disableSplitView = function (splitViewListItem) {
+
+		if ($('#pages-controller-cms-content').width() < 1340 && $('.cms-preview').width() < 1340 && !splitViewListItem.hasClass('result-selected')) {
```

Comment:
Can you make this a bit more soft-coded? check out LeftAndMain.js / LayoutOptions for an example of this being done right. :) There is a minPreviewWidth already.

LeftAndMain.Layout.js is where this logic seems to be invoked (`obj.layout`). We should keep all layout logic logic like this in the same place shouldn't we? :P